server/graphql/v2/interface/Account.js

```diff
@@ -259,6 +259,10 @@ const accountFieldsDefinition = () => ({
     type: Location,
     description: 'The address associated to this account. This field is always public for collectives and events.',
   },
+  categories: {
+    type: new GraphQLList(GraphQLString),
```

Comment:
We can probably do that since there's a default

```suggestion
    type: new GraphQLNonNull(new GraphQLList(GraphQLString)),
```